.github/workflows/main.yml

```diff
@@ -1,6 +1,12 @@
-name: Test and lint
```

Comment:
I think the previous name is more explicit.

Reply:
This name is the prefix for the jobs. Before it was shown as:
`Test and lint / lint` and it was confusing to see the same job twice. You'll still see lint and test in the jobs.